Resources/Prototypes/Entities/Objects/Weapons/Guns/Basic/watergun.yml

```diff
@@ -84,7 +121,7 @@
   - type: SolutionContainerManager
     solutions:
       chamber:
-        maxVol: 100 #10 shots
+        maxVol: 2500 #500 shots
```

Comment:
Why is this changed to be so high? Something like 500 would make more sense if its' intended to be an upgrade.

Reply:
Agreed.
Changed to 500u max volume (100 shots).